static/js/containers/SettingsPage.js

```diff
@@ -0,0 +1,100 @@
+/* global SETTINGS: false */
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Jumbotron from '../components/Jumbotron';
+import PrivacyForm from '../components/PrivacyForm';
+import ProfileFormContainer from './ProfileFormContainer'
+import {
+  startProfileEdit,
+  updateProfile,
+  updateProfileValidation,
+  clearProfileEdit,
+  saveProfile,
+} from '../actions';
+
+class SettingsPage extends React.Component {
+
+  static propTypes = {
+    dispatch:       React.PropTypes.func.isRequired,
+    updateProfile:  React.PropTypes.func.isRequired,
+    saveProfile:    React.PropTypes.func.isRequired,
+    userProfile:    React.PropTypes.object.isRequired,
+    ui:             React.PropTypes.object.isRequired,
+    errors:         React.PropTypes.object,
+    isSettingPage:  React.PropTypes.bool
+  };
+
+  constructor(props) {
+    super(props);
+    this.updateProfile = this.updateProfile.bind(this);
+    this.saveProfile = this.saveProfile.bind(this);
```

Comment:
instead of declaring these methods below and binding them to `this` in the constructor you can just use the class property pattern instead:

```js
class SettingsPage extends React.Component {
  updateProfile = profile => {
    ...
  };
}
```

if you declare a class property as an arrow function `this` will already be bound to the instance at run-time (it will do what you expect, basically).